src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        final int number = random.nextInt(100);
```

Comment:
this is still the magic number. let's make it a constant for the class

Reply:
Not fixed, do not be shy to ask a question in your PErsonal channel or in the questions chat, if you do not understand what are we commenting about.
We are talking about the `100` it should be a constant with some name, because on the big scale lots of such a numbers in a code make it unreadable